ml-proto/test/block.wast

```diff
@@ -46,43 +46,62 @@
     (f32.gt (block (call $nop) (f32.const 3)) (block (call $nop) (f32.const 3)))
   )
 
-  (func "br-bare" (result i32)
+  (func "break-bare" (result i32)
```

Comment:
Ok. I guess I'm already in the habit of thinking in terms of https://github.com/WebAssembly/design/pull/652. If we do that, we can simplify these tests.